cubeviz/image_viewer.py

```diff
@@ -21,7 +22,8 @@
 
 from .utils.contour import ContourSettings
 
-import matplotlib.image as mimage
+DEFAULT_NUMBER_OF_CONTOUR_LEVELS = 8
```

Comment:
Could you change these two variables to be:
CONTOUR_DEFAULT_NUMBER_OF_LEVELS
CONTOUR_MAX_NUMBER_OF_LEVELS

Given this code isn't living in a contour package or file I want to have the descriptor for what these relate to right at the beginning.  

Works fine though otherwise.